### ISSUE TITLE
Styled-components doesn't add vendor prefixes

### ISSUE THREAD

**Issue by kirillgenets** [styled-components/styled-components#3011]
### System:
 - OS: Linux 4.4 Ubuntu 18.04.2 LTS (Bionic Beaver)     
 - CPU: (8) x64 Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz
 - Memory: 802.72 MB / 7.91 GB
 - Container: Yes
 - Shell: 4.4.19 - /bin/bash
### Binaries:
 - Node: 10.15.3 - ~/.nvm/versions/node/v10.15.3/bin/node
 - Yarn: 1.22.0 - ~/.yarn/bin/yarn
 - npm: 6.4.1 - ~/.nvm/versions/node/v10.15.3/bin/npm
### npmPackages:
 - babel-plugin-styled-components: ^1.10.7 => 1.10.7
 - styled-components: ^5.0.1 => 5.0.1 

## Reproduction

I expected that styled-components will automatically add vendor-prefixes, but it really doesn't.
Here is my babel.config.js:
```
module.exports = (api) => {
	api.cache(true);

	return {
		presets: [
			[
				'@babel/preset-env',
				{
					useBuiltIns: 'usage',
					corejs: 3,
				},
			],
			'@babel/preset-typescript',
		],
		plugins: [
			'babel-plugin-styled-components',
			'@babel/plugin-transform-modules-commonjs',
		],
	};
};
```
Here is example of usage styled-components:
```
import styled from 'styled-components';
import { WrapperProps } from './types';

export const Wrapper = styled.div`
	display: grid;
	box-sizing: border-box;
	border-top-style: solid;
	border-top-width: ${(props: WrapperProps) => props.theme.weight};
	border-top-color: ${(props: WrapperProps) => props.theme.color};
	width: ${(props: WrapperProps) => props.length};
	grid-template-areas: ${(props: WrapperProps) => props.gridTemplateAreas};
	grid-template-columns: ${(props: WrapperProps) => props.gridTemplateColumns};
	grid-template-rows: ${(props: WrapperProps) => props.gridTemplateRows};
`;
```
Here is my .browserslistrc file:
```
last 2 Chrome versions
last 2 Firefox versions
last 2 Safari versions
last 2 Opera versions
last 2 Edge versions
last 2 iOS versions
last 2 ChromeAndroid versions
IE 11
IE >= 8
```

## Expected Behavior
To add vendor prefixes for all browsers from .browserslistrc

## Actual Behavior
It doesn't.
That's what I see in IE:
![image](https://user-images.githubusercontent.com/40032777/74105006-18ed2500-4b6b-11ea-831a-a03764ea1d0b.png)
That's a bundle file of this component (after babel compilation):
```
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.Wrapper = void 0;

var _styledComponents = _interopRequireDefault(require("styled-components"));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

var Wrapper = _styledComponents["default"].div.withConfig({
  displayName: "elements__Wrapper",
  componentId: "w3d13y-0"
})(["display:grid;box-sizing:border-box;border-top-style:solid;border-top-width:", ";border-top-color:", ";width:", ";grid-template-areas:", ";grid-template-columns:", ";grid-template-rows:", ";"], function (props) {
  return props.theme.weight;
}, function (props) {
  return props.theme.color;
}, function (props) {
  return props.length;
}, function (props) {
  return props.gridTemplateAreas;
}, function (props) {
  return props.gridTemplateColumns;
}, function (props) {
  return props.gridTemplateRows;
});

exports.Wrapper = Wrapper;
```

**Comment by probablyup** [comment]
See https://github.com/thysultan/stylis.js/issues/119, thanks.

tl;dr it can't be effectively auto-prefixed due to needing knowledge of how many grid items/rows are in use, but there is a rudimentary plugin in the thread you could set up
